x/liquidfarming/keeper/auction.go

```diff
@@ -169,9 +172,10 @@ func (k Keeper) getNextAuctionIdWithUpdate(ctx sdk.Context, poolId uint64) uint6
 	return auctionId
 }
 
-func (k Keeper) skipRewardsAuction(ctx sdk.Context, rewards sdk.Coins, auction types.RewardsAuction) {
+func (k Keeper) skipRewardsAuction(ctx sdk.Context, rewards sdk.Coins, feeRate sdk.Dec, auction types.RewardsAuction) {
 	auction.SetRewards(rewards)
 	auction.SetStatus(types.AuctionStatusSkipped)
+	auction.SetFeeRate(feeRate)
```

Comment:
Does the `winning_amount` include `fee`? Or, `winning_amount` is how much coins are sent to the auction winner?

Reply:
`winning_amount` is the winning bid amount that the bidder has placed and fees have to do with rewards.